apps/reference/docs/guides/auth/auth-azure.mdx

```diff
@@ -69,14 +69,6 @@ This will allow your users to use your custom Azure login page when logging in.
 
 The JavaScript client code is documented in the [Supabase OAuth Reference](/docs/reference/javascript/auth-signin#sign-in-using-third-party-providers).
```

Comment:
1. The parameters section for the signIn ref docs aren't very helpful right now: https://supabase.com/docs/reference/javascript/auth-signin#parameters
2. Let's have the callout text be the paragraph above the code example.

```suggestion
Supabase Auth requires a valid email address from Azure. Call [signIn](https://supabase.com/docs/reference/javascript/auth-signin) and include the `email` scope.
```

Reply:
Was handled by another docs update.